llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -675,48 +675,6 @@ class VPBlockBase {
   virtual VPBlockBase *clone() = 0;
 };
 
-/// A value that is used outside the VPlan. The operand of the user needs to be
-/// added to the associated phi node. The incoming block from VPlan is
-/// determined by where the VPValue is defined: if it is defined by a recipe
-/// outside a region, its parent block is used, otherwise the middle block is
-/// used.
-class VPLiveOut : public VPUser {
```

Comment:
Note that with VPLiveOut replaced by VPIRInstructions, all VPUsers are recipes.

Reply:
Yep, removed `LiveOut` VPUserID, but retained the enum for now (may be helpful for marking VPBlocks/Regions as users (e.g. of VF, TC). But can also drop it all and add back if/when needed.

Reply:
Dropping seems cleaner to me.